### PR TITLE
use custom operator

### PR DESCRIPTION
**Pull request by hguerrero**
##### SUMMARY

Update the application of the instructions to v0.0.63 and use the custom operator

Fixes https://github.com/RedHat-Middleware-Workshops/dayinthelife-streaming/issues/40 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

ocp4-workload-dil-streaming

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: ansible/roles/ocp4-workload-dil-streaming/defaults/main.yaml [hunk 1]
```diff
@@ -21,14 +21,15 @@ mssqlserver_image: microsoft/mssql-server-linux
...
 webapp_client_id: tutorial-web-app
+ocp4_workload_dil_serverless_webapp_group_name: dedicated-admins
 webapp_operator_template_path: /home/tutorial-web-app-operator/deploy/template/tutorial-web-app.yml
 webapp_operator_resources: 'https://github.com/integr8ly/tutorial-web-app-operator/archive/v{{webapp_operator_release_tag}}.zip'
 webapp_operator_resource_items:
   - rbac.yaml
   - sa.yaml
   - crd.yaml
-  - operator.yaml
+  # - operator.yaml
```
Comment (stencell) [inline ansible/roles/ocp4-workload-dil-streaming/defaults/main.yaml:33]: Please remove this vs leaving it commented out.

File: ansible/roles/ocp4-workload-dil-streaming/templates/instructions-group.yaml.j2 [hunk 2]
```diff
@@ -0,0 +1,6 @@
+kind: Group
+apiVersion: user.openshift.io/v1
+metadata:
+  name: '{{ocp4_workload_dil_serverless_webapp_group_name}}'
```
Comment (stencell) [inline ansible/roles/ocp4-workload-dil-streaming/templates/instructions-group.yaml.j2:4]: Please add space around the `{{` here and a couple of lines down

**Review by stencell — CHANGES_REQUESTED** [review]
few comments to address, please

**Comment by hguerrero** [comment]
addressed the comments in PR

File: ansible/roles/ocp4-workload-dil-streaming/defaults/main.yaml (hunk 1 above)
Comment (hguerrero) [reply ansible/roles/ocp4-workload-dil-streaming/defaults/main.yaml]: done

File: ansible/roles/ocp4-workload-dil-streaming/tasks/provision_webapp.yaml [hunk 3]
```diff
@@ -28,6 +28,12 @@
     src: '{{ webapp_operator_tmp }}/tutorial-web-app-operator-{{ webapp_operator_release_tag }}/deploy/{{ item }}'
   loop: "{{ webapp_operator_resource_items }}"
 
+- name: Deploy custom Operator
+  k8s:
+    state: present
+    namespace: '{{ webapp_namespace }}'
+    resource_definition: '{{ lookup("url", "https://gist.githubusercontent.com/hguerrero/6bde163c3244a3a3ebc7d044f3c810f7/raw/d2676b2d6772c510aa038b4645de78076099eb54/operator.yaml", split_lines=False) | from_yaml_all}}'
```
[the remark this answers is not in the document]
Comment (hguerrero) [reply ansible/roles/ocp4-workload-dil-streaming/tasks/provision_webapp.yaml]: using templates now

File: ansible/roles/ocp4-workload-dil-streaming/templates/instructions-group.yaml.j2 (hunk 2 above)
Comment (hguerrero) [reply ansible/roles/ocp4-workload-dil-streaming/templates/instructions-group.yaml.j2]: done, also fix workload name

**Review by stencell — APPROVED** [review]

**Pull request merged**